cfme/v2v/infrastructure_mapping.py

```diff
@@ -0,0 +1,401 @@
+import attr
+
+from navmazing import NavigateToAttribute, NavigateToSibling
+from widgetastic.widget import Checkbox, ConditionalSwitchableView, View
+from widgetastic_manageiq import (
+    InfraMappingList,
+    InfraMappingTreeView,
+    MultiSelectList,
+    V2vPaginatorPane,
+)
+from widgetastic_patternfly import (
+    AggregateStatusCard,
+    BootstrapSelect,
+    Button,
+    Dropdown,
+    SelectorDropdown,
+    Text,
+    TextInput,
+)
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from cfme.utils.update import Updateable
+from cfme.utils.version import Version, VersionPicker
+from cfme.v2v.migration_plans import MigrationPlanView
+
+
+class InfraMappingWizardCommon(View):
+    """Infrastructure Mapping common View"""
+    add_mapping = Button("Add Mapping")
+    remove_mapping = Button("Remove Selected")
+    remove_all_mappings = Button("Remove All")
+    mappings_tree = InfraMappingTreeView(tree_class="treeview")
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
+    """All infrastructure mapping View"""
+
+    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
+    create_infra_mapping = Text(locator='(//a|//button)[text()="Create Infrastructure Mapping"]')
+    configure_providers = Text(locator='//a[text()="Configure Providers"]')
+    paginator_view = View.include(V2vPaginatorPane, use_parent=True)
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    sort_order = Text(".//button[./span[contains(@class,'sort-direction')]]")
+    filter_by_dropdown = SelectorDropdown("id", "filterFieldTypeMenu")
+    sort_by_dropdown = SelectorDropdown("id", "sortTypeMenu")
+
+    @property
+    def is_displayed(self):
+        # TODO: Resmove 1st condition, once /manageiq-v2v/issues/768 fix is backported to 510z
```

Comment:
This is fixed https://github.com/manageiq/manageiq-v2v/issues/768 so we need to remove this comment.